views/template/index.pug

```diff
@@ -0,0 +1,27 @@
+extends layout.pug
+
+block head
+	.container
+		.logo
+			img.head-logo(src="/assets/img/phpbblogo.png")
+			h1 Styleguide
+			img.icon(src="/assets/img/github_icon.png" alt="github")
+		.content
+			p A Styleguide Driven Development System for creating, managaing, customizing, & editing themes for phpBB.
+			p GSoC 2017 project being developed by Sahil Khokhar.
+		.home_btn
+			- var btnType = "info"
+			- var btnSize = "lg"
```

Comment:
These are the attributes that we can use inside a button tag. I've removed it for now.